src/scripts.js

```diff
@@ -106,6 +109,55 @@ getAllData().then(responses => {
   displayAllNames();
 });
 
+
+//~~~~~~~~~Post Request~~~~~~~~~~~~~~
+const addNewIngredient = (dataType, newIngredient) => {
+  fetch(`http://localhost:3001/api/v1/${dataType}`, {
+    method: "POST",
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(newIngredient)
+  })
+  .then(response => {
+    console.log("Hello,POST, response" .json())
+    if(!response.ok){
+      throw new Error(response.statusText)
+     } else {
+       return response.json()
+     }   
+   })
+   .then(ingredient => addIngredientToPage(ingredient))
+   .catch(err => err  (alert(`MUST FILL OUT ALL BOXES!!!`)))// What can be done better here?
+ }
+
+ getAllData(); //from fetch all data function 
+ const addIngredientToPage = ingredients => {
```

Comment:
Still confused about this function and if it is necessary... but ok to merge and see if we can tie it all together